APIs/SweccAPI.py

```diff
@@ -22,3 +22,21 @@ def auth(self, discord_username, id, username):
 
         response = requests.put(f"{self.url}/members/verify-discord/", headers=headers, json=data)
         return response.status_code
+
+    def leetcode_leaderboard(self, order_by=None):
+        logging.info("Fetching leetcode leaderboard order by %s", order_by)
+        headers = {
```

Comment:
Let's create self.headers that stores this since it will be the same for all the sweccAPI logic. Then we can just use self.headers in both auth and leetcode calls.

Reply:
done